core/templates/dev/head/app.js

```diff
@@ -355,6 +355,62 @@ oppia.factory('windowDimensionsService', ['$window', function($window) {
   };
 }]);
 
+// Service for sending events to Google Analytics.
+//
+// Note that events are only sent if the CAN_SEND_ANALYTICS_EVENTS flag is
+// turned on. This flag must be turned on explicitly by the application
+// owner.
+oppia.factory('siteAnalyticsService', ['$window', function($window) {
+  var CAN_SEND_ANALYTICS_EVENTS = GLOBALS.CAN_SEND_ANALYTICS_EVENTS;
+
+  // For definitions of the various arguments, please see:
+  // developers.google.com/analytics/devguides/collection/analyticsjs/events
+  var _sendEventToGa = function(eventCategory, eventAction, eventLabel) {
```

Comment:
I don't think Ga is a standard acronym; I would just say _sendEventToGoogleAnalytics